lib/premailer/rails/css_loaders/file_system_loader.rb

```diff
@@ -6,8 +6,15 @@ module FileSystemLoader
 
         def load(url)
           path = URI(url).path
-          file_path = "public#{path}"
-          File.read(file_path) if File.exist?(file_path)
+
+          # Remove leading slash if it exists
+          path = path[1..-1] if path[0,1] == "/"
+
+          file_path = ::Rails.root.join("public/#{path}").to_s
```

Comment:
The path needs to be relative to Rails root and not the current working directory. This wasn't working on some environments where the process is started outside of the app folder.